recipes/agate-excel/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{%set name = "agate-excel" %}
+{%set version = "0.2.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "5e75452fefe1ffe18fe695eeb08337c50eb3f0b74bfe108d5a85a2bb3ee806d2" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py>35]
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - agate >=1.5.0
```

Comment:
Can you try removing the `run` time dependencies from the `build` section and see how it goes?